Solution1/Solution1.Tests/BL/Validators/ForecastWeatherRequestDTOValidatorTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Vlidators;
+using FluentValidation;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Weather.Tests.BL.Validators
+{
+    public class ForecastWeatherRequestDTOValidatorTests
+    {
+        private readonly IValidator<ForecastWeatherRequestDTO> _validator;
+
+        public ForecastWeatherRequestDTOValidatorTests()
+        {
+            _validator = new ForecastWeatherRequestDTOValidator();
+        }
+
+        [Theory]
+        [InlineData("Minsk", 3, true, 0)]
+        [InlineData("Minsk", -1, false, 1)]
+        [InlineData(default, 3, false, 1)]
+        [InlineData("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", 3, false, 1)]
+        [InlineData(default, 6, false, 2)]        
```

Comment:
where are checks on days number??

Reply:
you have 2 rules in validator, why you check only one?

Reply:
*